modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/AspNetCoreServerCodegen.java

```diff
@@ -362,8 +409,8 @@ private void setClassModifier() {
             operationModifier.setOptValue(classModifier.getOptValue());
             additionalProperties.put(OPERATION_MODIFIER, operationModifier.getOptValue());
             LOGGER.warn("classModifier is " + classModifier.getOptValue() + " so forcing operatonModifier to " + operationModifier.getOptValue());
-        } else {
-            setCliOption(operationModifier);
+        //} else {
+            //setCliOption(operationModifier);
```

Comment:
@A-Joshi minor suggestion: what about completely remove these 2 lines of commented code?

Reply:
Removed.